client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -155,21 +157,40 @@ class PodcastingDetails extends Component {
 			handleSubmitForm,
 			siteSlug,
 			siteId,
+			isPrivate,
 			podcastingCategoryId,
 			translate,
 			isPodcastingEnabled,
 			fields,
 		} = this.props;
+
 		if ( ! siteId ) {
 			return null;
 		}
 
+		const writingHref = `/settings/writing/${ siteSlug }`;
+
+		if ( isPrivate ) {
+			return (
+				<div
+					className="main main-column" // eslint-disable-line
+					role="main"
+				>
```

Comment:
The `render` method can probably be refactored a bit to avoid duplication of the wrapper components, but I'd propose that we wait until we have a couple more error messages in subsequent PRs.  This way we can be sure the refactoring will work for multiple situations.